assets/js/base/context/cart-checkout/payment-methods/use-payment-method-registration.js

```diff
@@ -24,7 +24,9 @@ import {
  */
 const handleRegistrationError = ( error ) => {
 	if ( CURRENT_USER_IS_ADMIN ) {
-		throw new Error(
+		// This is only shown to admin users, so it's ok to print an error to the browser console.
+		// eslint-disable-next-line no-console
+		console.error(
```

Comment:
Seems like we have a few lines of code just to throw up a `console.error` – is it worth it? 

[Later in the file, there's a comment about this info being presented as a notice.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/5c5bd3a70c7d96208c5f975cfc2864f9d665d6c7/assets/js/base/context/cart-checkout/payment-methods/use-payment-method-registration.js#L126) That makes sense to me – if we're going to add explicit support for admins of misconfigured sites, we might as well use notice to show in the UI.